packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hello-world-cjs",
+  "version": "0.0.1",
+  "description": "An example vanilla that uses cjs modules that works with Honeycomb's OpenTelemetry Web Distro",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "build": "cd $npm_config_local_prefix/../../ && npm run build && cd $npm_config_local_prefix &&  mkdir -p ./dist/umd/ && cp -vR $npm_config_local_prefix/../../dist/umd/index.js ./dist/umd",
```

Comment:
I noticed none of our other examples actually build the parent project (unless you do the docker image). Should we cut out the build part for consistency? Would make the command a lot shorter too.

Reply:
I agree. I like how this is turn-key 😸  but it does introduce a variant pattern. 😿. 
I def want to standardize how this work so it works in a predictable way. But I need for expressive tooling.How do you feel about me addressing this more wholistically in a follow up PR?

Reply:
That sounds great! I definitely agree that having to  do the builds individually is not the best.